### PR TITLE
migrate code from googleapis/python-videointelligence

### PR DESCRIPTION
**Pull request by dizcology**
migrate code from python-videointelligence

- Adds tutorials using Cloud Client [(#930)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/930)
- Adds new examples, replaces markdown with restructured text [(#945)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/945)
- Fix README rst links [(#962)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/962)
- updated the gcs files used in Usage [(#960)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/960)
- Updates examples for video [(#968)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/968)
- Fixes broken example in docs [(#987)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/987)
- Places format calls inside print function calls. [(#995)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/995)
- Auto-update dependencies. [(#1093)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1093)
- Update all generated readme auth instructions [(#1121)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1121)
- Auto-update dependencies. [(#1123)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1123)
- Video v1beta2 [(#1088)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1088)
- Added Link to Python Setup Guide [(#1158)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1158)
- Auto-update dependencies. [(#1186)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1186)
- update samples to v1 [(#1221)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1221)
- Do not base64 encode the input content. [(#1241)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1241)
- remove empty input_uri [(#1244)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1244)
- Disable another FACE_DETECTION test [(#1253)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1253)
- Added "Open in Cloud Shell" buttons to README files [(#1254)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1254)
- Auto-update dependencies. [(#1377)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1377)
- Remove video face [(#1348)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1348)
- Face detection beta features [(#1414)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1414)
- [DO NOT MERGE] remove face detection samples [(#1435)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1435)
- Regenerate the README files and fix the Open in Cloud Shell link for some samples [(#1441)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1441)
- update client library version [(#1442)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1442)
- Update READMEs to fix numbering and add git clone [(#1464)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1464)
- Add video region tags [(#1489)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1489)
- Add automatic punctuation to video speech transcription [(#1603)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1603)
- Video Intelligence region tag update [(#1639)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1639)
- Use explicit URIs for Video Intelligence sample tests [(#1743)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1743)
- Access Display Names of enum fields via enum object [(#1738)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1738)
- Add beta snippets for object tracking / text detection [(#1773)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1773)
- more info on alternatives [(#1791)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1791)
- feat: video speech transcription [(#1849)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1849)
- Lib version update for Video Intelligence API [(#1850)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1850)
- Video: edit region tag [(#1851)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1851)
- Auto-update dependencies. [(#1980)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1980)
- Video GA text detection and object tracking [(#2024)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2024)
- add video streaming samples [(#2047)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2047)
- Wait longer for sometimes slow operation [(#2056)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2056)
- Fix urllib2 [(#2059)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2059)
- replace demomaker with cloud-samples-data/video for video intelligenc… [(#2162)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2162)
- fix python2 print unicode issue [(#2183)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2183)
- explicitly set timeout in steaming_annotate_video, and comment [(#2182)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2182)
- update streaming samples [(#2215)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2215)
- [video] Streaming AutoML Classification [(#2313)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2313)
- Use video_path fixture [(#2363)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2363)
- Adds updates for samples profiler ... vision [(#2439)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2439)
- Auto-update dependencies. [(#2005)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2005)
- video: add auto-generated Logo Recognition Samples [(#2755)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2755)
- video: adding face detection and person detection samples for beta [(#2919)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2919)
- chore(deps): update dependency google-cloud-storage to v1.26.0 [(#3046)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3046)
- fix: wait up to 20 seconds for blob to appear [(#3061)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3061)
- Update analyze.py [(#3088)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3088)
- video: adding GA samples for logo detection [(#3098)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3098)
- Simplify noxfile setup. [(#2806)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2806)
- video: fix flaky beta tests [(#3235)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3235)
- video: use flaky on flaky timeouts [(#3385)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3385)
- vision: delete obsolete beta snippets for logo detection  [(#3114)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3114)
- [video] testing: mark some tests witih flaky [(#3446)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3446)
- chore(deps): update dependency google-cloud-storage to v1.28.0 [(#3260)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3260)
- chore: some lint fixes [(#3751)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3751)
- chore(deps): update dependency google-cloud-storage to v1.28.1 [(#3785)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3785)
- testing: start using btlr [(#3959)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3959)
- fix: changes positional to named pararameters in Video samples [(#4017)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4017)
- Replace GCLOUD_PROJECT with GOOGLE_CLOUD_PROJECT. [(#4022)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4022)
- fix: removes reference to 'facial hair' from video face samples [(#4052)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4052)
- fix: changes a positional to named arg [(#4092)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4092)
- Update dependency google-cloud-videointelligence to v1.15.0 [(#4041)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4041)
- chore(deps): update dependency google-cloud-storage to v1.29.0 [(#4040)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4040)
- chore: changes boilerplate to LLC [(#4201)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4201)
- chore(deps): update dependency pytest to v5.4.3 [(#4279)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4279)
- Update dependency flaky to v3.7.0 [(#4300)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4300)
- testing(videointelligence): retry harder upon 409s [(#4361)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4361)
- Update dependency google-cloud-storage to v1.30.0
- Update dependency pytest to v6 [(#4390)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4390)
- chore(deps): update dependency google-cloud-storage to v1.31.0 [(#4564)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4564)
- chore(deps): update dependency google-cloud-storage to v1.31.1 [(#4714)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4714)
- chore(deps): update dependency google-cloud-storage to v1.31.2 [(#4750)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4750)
- chore: update templates
- chore: flaky detect_faces tests in beta (#58)
- docs: add samples from video/cloud-client/labels (#49)
- samples: fixed flaky test on vision beta samples (#62)
- chore: fixed flaky test by adding retry (#66)
- chore(deps): update dependency google-cloud-videointelligence to v1.16.0 (#65)
- Adds quickstart [(#964)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/964)
- Auto-update dependencies. [(#1093)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1093)
- Update all generated readme auth instructions [(#1121)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1121)
- Auto-update dependencies. [(#1123)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1123)
- Video v1beta2 [(#1088)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1088)
- Added Link to Python Setup Guide [(#1158)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1158)
- Tweak doc/help strings for sample tools  [(#1160)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1160)
- Auto-update dependencies. [(#1186)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1186)
- update samples to v1 [(#1221)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1221)
- Added "Open in Cloud Shell" buttons to README files [(#1254)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1254)
- Auto-update dependencies. [(#1377)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1377)
- Auto-update dependencies.
- Regenerate the README files and fix the Open in Cloud Shell link for some samples [(#1441)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1441)
- Update READMEs to fix numbering and add git clone [(#1464)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1464)
- Video Intelligence region tag update [(#1639)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1639)
- Auto-update dependencies. [(#1658)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1658)
- Auto-update dependencies. [(#1846)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1846)
- Longer timeouts to address intermittent failures [(#1871)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1871)
- Auto-update dependencies. [(#1980)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1980)
- replace demomaker with cloud-samples-data/video for video intelligenc… [(#2162)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2162)
- Adds updates for samples profiler ... vision [(#2439)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2439)
- Auto-update dependencies. [(#2005)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2005)
- chore(deps): update dependency google-cloud-videointelligence to v1.14.0 [(#3169)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3169)
- Simplify noxfile setup. [(#2806)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2806)
- Update dependency google-cloud-videointelligence to v1.15.0 [(#4041)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4041)
- chore(deps): update dependency pytest to v5.4.3 [(#4279)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4279)
- Update dependency pytest to v6 [(#4390)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4390)
- chore(deps): update dependency google-cloud-videointelligence to v1.16.0 [(#4798)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4798)
- chore(deps): update dependency pytest to v6.1.1 [(#4761)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4761)
- chore(deps): update dependency pytest to v6.1.2 [(#4921)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4921)
- chore: updates templates
- chore: lints and blackens
- chore(deps): update dependency google-cloud-storage to v1.32.0 (#64)
- docs: add samples from video/cloud-client/shotchange (#72)
- chore(deps): update dependency google-cloud-storage to v1.33.0 (#75)
- feat!: use microgenerator (#76)
- chore: added retry for flaky test (#84)
- feat: updates person detection samples for GA (#86)
- chore: another flaky test for video beta samples (#91)
- chore(deps): update dependency google-cloud-videointelligence to v1.16.1 (#82)
- chore(deps): update dependency google-cloud-storage to v1.35.0 (#92)
- feat: Introduce tracks and thumbnails fields for FaceDetectionAnnotations (#90)
- chore: update templates (#95)
- chore(deps): update dependency google-cloud-videointelligence to v2 (#83)
- chore: increased timeout for flaky service & added retry (#103)
- chore(deps): update dependency google-cloud-storage to v1.36.0 (#107)
- chore(deps): update dependency google-cloud-storage to v1.36.1 (#109)
- chore(deps): update dependency google-cloud-storage to v1.36.2 (#112)
- chore: increased designed timeout by 1 minute (#110)
- chore: added retry on flaky video service (#115)
- chore: updating test environment (#116)
- chore(deps): update dependency google-cloud-storage to v1.37.0 (#119)
- samples: updates face recognition to v1 (#127)
- chore: add license headers (#129)
- chore: Refactor test (#131)
- chore: migrate to owl bot (#135)
- chore(deps): update dependency pytest to v6.2.4 (#142)
- chore(deps): update dependency google-cloud-storage to v1.38.0 (#128)
- chore(deps): update dependency google-cloud-videointelligence to v2.1.0 (#125)
- chore: increased timeout by 200s to reduce flakiness (#152)
- samples: add streaming_automl_object_tracking sample (#143)
- chore: new owl bot post processor docker image (#156)
- chore(deps): update dependency google-cloud-videointelligence to v2.2.0 (#159)
- chore: increased timeout to max (#161)
- chore(deps): update dependency google-cloud-storage to v1.39.0 (#169)
- chore(deps): update dependency google-cloud-storage to v1.40.0 (#174)
- chore(deps): update dependency backoff to v1.11.0 (#180)
- chore(deps): update dependency google-cloud-videointelligence to v2.3.1 (#181)
- docs: add Samples section to CONTRIBUTING.rst (#190)
- chore(deps): update dependency backoff to v1.11.1 (#186)
- chore(deps): update dependency google-cloud-storage to v1.41.1 (#185)
- chore(deps): update dependency google-cloud-videointelligence to v2.3.2 (#195)
- chore(deps): update dependency google-cloud-storage to v1.42.0 (#200)
- chore: fix INSTALL_LIBRARY_FROM_SOURCE in noxfile.py (#198)
- chore: drop mention of Python 2.7 from templates (#202)
- chore: fix flaky sample track_objects (#211)
- chore(deps): update dependency pytest to v6.2.5 (#209)
- chore(samples): drop obsolete samples (#210)
- samples: add streaming_automl_action_recognition sample (#215)
- chore(deps): update dependency google-cloud-storage to v1.42.1 (#217)
- chore: blacken samples noxfile template (#218)
- chore(deps): update dependency google-cloud-storage to v1.42.2 (#219)
- chore(deps): update dependency google-cloud-videointelligence to v2.3.3 (#224)
- chore(deps): update dependency google-cloud-storage to v1.42.3 (#227)
- chore: fail samples nox session if python version is missing (#226)
- chore(deps): update dependency google-cloud-videointelligence to v2.4.0 (#231)
- chore(python): Add kokoro configs for python 3.10 samples testing (#233)
- chore(deps): update dependency google-cloud-videointelligence to v2.5.0 (#239)
- chore(deps): update dependency google-cloud-videointelligence to v2.5.1 (#244)
- chore(python): run blacken session for all directories with a noxfile (#247)
- chore(deps): update dependency google-cloud-storage to v1.43.0 (#251)
- chore(samples): Add check for tests in directory (#259)
- chore(deps): update dependency google-cloud-storage to v1.44.0 (#256)
- chore(deps): update dependency google-cloud-storage to v2 (#262)
- chore(python): Noxfile recognizes that tests can live in a folder (#264)
- chore(deps): update dependency google-cloud-storage to v2.1.0 (#265)
- chore(deps): update all dependencies (#276)
- chore: Adding support for pytest-xdist and pytest-parallel (#283)
- chore(deps): update dependency google-cloud-videointelligence to v2.6.1 (#286)
- chore(deps): update dependency pytest to v7.1.0 (#288)
- chore(deps): update dependency google-cloud-storage to v2.2.0 (#289)
- chore(deps): update dependency google-cloud-storage to v2.2.1 (#290)
- chore(deps): update dependency pytest to v7.1.1 (#291)
- chore(python): use black==22.3.0 (#295)
- chore(deps): update all dependencies (#309)
- chore(python): add nox session to sort python imports (#312)
- chore(deps): update dependency pytest to v7.1.2 (#316)
- chore: changed model_id for test_action_rec (#315)
- chore(deps): update dependency backoff to v2 (#317)
- chore(deps): update dependency backoff to v2.0.1 (#319)
- chore(deps): update dependency google-cloud-videointelligence to v2.7.0 (#321)
- fix: require python 3.7+ (#339)
- chore(deps): update all dependencies (#331)
- chore(deps): update all dependencies (#346)
- chore(deps): update all dependencies (#348)
- chore(deps): update dependency google-cloud-videointelligence to v2.8.1 (#354)
- chore(deps): update dependency pytest to v7.1.3 (#365)
- chore: detect samples tests in nested directories (#369)
- chore(deps): update dependency google-cloud-videointelligence to v2.8.2 (#373)
- chore(deps): update dependency backoff to v2.2.1 (#374)
- chore(deps): update dependency google-cloud-videointelligence to v2.8.3 (#377)
- chore(deps): update dependency pytest to v7.2.0 (#378)
- chore(deps): update dependency google-cloud-storage to v2.6.0 (#381)
- chore(python): drop flake8-import-order in samples noxfile (#385)
- test: fix the streaming_automl_action_recognition test with an newly trained model (#391)
- chore(deps): update dependency google-cloud-storage to v2.7.0 (#389)
- chore(deps): update dependency google-cloud-videointelligence to v2.9.0 (#392)
- chore(python): add support for python 3.11 [autoapprove] (#393)
- chore(deps): update dependency google-cloud-videointelligence to v2.10.0 (#396)
- chore(deps): update dependency pytest to v7.2.1 (#397)
- chore(deps): update dependency google-cloud-videointelligence to v2.10.1 (#400)
- chore(deps): update dependency google-cloud-videointelligence to v2.11.0 (#409)
- chore(deps): update dependency pytest to v7.2.2 (#410)
- chore(deps): update all dependencies (#414)
- chore(deps): update dependency pytest to v7.3.1 (#416)
- fix: expand region tags to include `import io` (#418)
- build(deps): bump requests to 2.31.0 [autoapprove] (#424)
- chore(deps): update dependency google-cloud-storage to v2.9.0 (#422)
- chore(deps): update dependency google-cloud-videointelligence to v2.11.2 (#425)
- chore(deps): update dependency pytest to v7.3.2 (#428)
- chore(deps): update all dependencies (#429)
- chore(deps): update dependency google-cloud-videointelligence to v2.11.3 (#435)

## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by rsamborski — CHANGES_REQUESTED** [review]
Thank you for the code migration. Before merging please update:
 - [CODEOWNERs](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS)
 - [Blunderbuss](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml)

Couple additional notes to remember:
 - CLA issues will require admin merge
 - Remember to use `merge` instead of `squash and merge` to keep history intact

File: videointelligence/samples/analyze/analyze_test.py
Comment (dizcology) [reply videointelligence/samples/analyze/analyze_test.py]: Removed all `@pytest.mark.slow`.

**Comment by dizcology** [comment]
CODEOWNERS and blunderbuss updated.

**Review by rsamborski — APPROVED** [review]

**Review by leahecole — APPROVED** [review]

**Comment by leahecole** [comment]
CLA is okay, these are xooglers

**Pull request merged**
